test/visual/anchor-tabs.html

```diff
@@ -1,26 +1,27 @@
 <!DOCTYPE html>
-
+<html></html>
```

Comment:
Hm, this insta-closed `</html>` tag is intentional here, or accidental auto-complete?

Reply:
Thanks, fixed.